schema.sql

```diff
@@ -3,6 +3,7 @@
 CREATE TABLE animals(
     id serial,
     name varchar(20),
+    species varchar(20),
```

Comment:
* Kindly add a column species of type string to your animal's table the right way [Read here on how to add a column](https://www.w3schools.com/sql/sql_alter.asp)